<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LightModeConfig</class>
 <widget class="QWidget" name="LightModeConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>680</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="tipLabel">
   <property name="geometry">
    <rect>
     <x>23</x>
     <y>319</y>
     <width>200</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei UI Light</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255,34,76);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QToolButton" name="enterBtn">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>366</y>
     <width>135</width>
     <height>34</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QToolButton" name="closeBtn">
   <property name="geometry">
    <rect>
     <x>647</x>
     <y>16</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_version">
   <property name="geometry">
    <rect>
     <x>26</x>
     <y>114</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 14px;</string>
   </property>
   <property name="text">
    <string>v1.0.1</string>
   </property>
  </widget>
  <widget class="QToolButton" name="miniBtn">
   <property name="geometry">
    <rect>
     <x>617</x>
     <y>16</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>150</y>
     <width>78</width>
     <height>15</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>-1</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12px &quot;微软雅黑&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>light mode</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="portLineEdit">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>309</y>
     <width>135</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei UI Light</family>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>IBeamCursor</cursorShape>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::DefaultContextMenu</enum>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>50320</string>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Normal</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>230</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12px &quot;微软雅黑&quot;;
color:white;</string>
   </property>
   <property name="text">
    <string>远程节点IP</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>290</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12px &quot;微软雅黑&quot;;
color:white;</string>
   </property>
   <property name="text">
    <string>端口</string>
   </property>
  </widget>
  <widget class="QComboBox" name="ipComboBox">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>254</y>
     <width>135</width>
     <height>32</height>
    </rect>
   </property>
   <property name="editable">
    <bool>true</bool>
   </property>
   <item>
    <property name="text">
     <string>127.0.0.1</string>
    </property>
   </item>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
